targets/TARGET_STM/qspi_api.c

```diff
@@ -35,6 +25,14 @@
 #include "pinmap.h"
 #include "PeripheralPins.h"
 
+#include "mbed_trace.h"
```

Comment:
Unifying the "tracing/debug/printf" facilities was an item on my TODO list. Do you know what's the cost of enabling it for builds that didn't use it before? For various build profiles?